File: 프로그래머스/1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.js
function solution(phone_number) {
    // 전화번호 뒷 4자리를 제외한 부분을 '*'로 변환
    let maskedPart = '*'.repeat(phone_number.length - 4);
    
    // 뒷 4자리를 그대로 두고 합침
    let answer = maskedPart + phone_number.slice(-4);
    
    return answer;
}

File: 프로그래머스/1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.py
def solution(phone_number):
    # 전화번호 뒷 4자리를 제외한 부분을 '*'로 변환
    masked_part = '*' * (len(phone_number) - 4)
    
    # 뒷 4자리를 그대로 두고 합침
    answer = masked_part + phone_number[-4:]
    
    return answer